### ISSUE TITLE
add some linked list programs in c++ or python

### ISSUE THREAD

**Issue by Sanga-byte29** [Vssut-Robotics/Algorithms-and-Datastructures#2]

**Comment by AkshayGidwani405** [comment]
Hi @Sanga-byte29! I can code some good linked list problems in C++. Can you assign me?

**Comment by pranit9000** [comment]
can you assign this to me.

**Comment by subashpaul** [comment]
can i work on this?

**Comment by Sanga-byte29** [comment]
@AkshayGidwani405 yes plzz give me the code i will merge your pr but give me in 3 good programs whichs has been asked in interviews and put in a folder i will merge the pr of yours

**Comment by savi001** [comment]
i am new to open source and so  interesed to work on it ,
ca u please assign to me.
